v4.13/catalog/rhtas-operator/catalog.json

```diff
@@ -196,66 +196,6 @@
         {
             "name": "",
             "image": "registry.redhat.io/rhtas/rhtas-rhel9-operator@sha256:2d2a22420249890a64b42c663d57c4b5c6638e0597fe74b01b275ba915e53ea2"
-        },
```

Comment:
Some of these related images have been deleted, for this issue the only change made to the catalog.json file should be the addition of the olm.deprecations :)